src/test/java/org/radarbase/appserver/service/UserServiceTest.java

```diff
@@ -196,6 +195,7 @@ void updateUser() {
                         .setLanguage("es")
                         .setLastDelivered(enrolmentDate)
                         .setLastOpened(enrolmentDate)
+                        .setAttributes(new HashMap<>())
```

Comment:
will this give issues in production when there are no attributes?

Reply:
Oh do you mean when the attributes is null or when it's empty empty? I made this empty assuming that the MP always sends the attribute property as empty map.